js/shared/input.js

```diff
@@ -0,0 +1,123 @@
+// Copyright 2014-2015 runtime.js project authors
```

Comment:
Well, it needs to be in a root level folder (root level being `js`) because it's used in `js/service/shell/input-box.js` and `js/core/stdio/stdin.js`.